### PR TITLE
fix(TDI-46004): tEXist components and tXMLRPCInput dependency import.

### PR DESCRIPTION
**Pull request by dmytrogrygorenko**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-46004

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [X] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.localprovider/components/tEXistConnection/tEXistConnection_java.xml [hunk 1]
```diff
@@ -92,7 +92,7 @@ COMPATIBILITY="ALL"
        <IMPORT NAME="Java-xmlrpc-client" MODULE="xmlrpc-client-3.1.3.jar" MVN="mvn:org.apache.xmlrpc/xmlrpc-client/3.1.3" REQUIRED="true" />
        <IMPORT NAME="Java-xmlrpc-common" MODULE="xmlrpc-common-3.1.3.jar" MVN="mvn:org.apache.xmlrpc/xmlrpc-common/3.1.3" REQUIRED="true" />
        <IMPORT NAME="Java-log4j" MODULE="log4j-1.2.17.jar" MVN="mvn:log4j/log4j/1.2.17"  UrlPath="platform:/plugin/org.talend.libraries.apache/lib/log4j-1.2.17.jar" REQUIRED="true" BundleID="" />
-       <IMPORT NAME="Java-ws-commons-util" MODULE="ws-commons-util-1.0.2.jar" MVN="mvn:org.talend.libraries/ws-commons-util-1.0.2/6.0.0"  REQUIRED="true" />
+       <IMPORT NAME="Java-ws-commons-util" MODULE="ws-commons-util-1.0.2.jar" MVN="mvn:org.apache.ws/ws-commons-util/1.0.2" REQUIRED="true" />
```
Comment (mbasiuk-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tEXistConnection/tEXistConnection_java.xml:95]: this path doesn't exist in maven.

**Review by mbasiuk-talend — APPROVED** [review]
LGTM

File: main/plugins/org.talend.designer.components.localprovider/components/tEXistConnection/tEXistConnection_java.xml (hunk 1 above)
Comment (dmytrogrygorenko) [reply main/plugins/org.talend.designer.components.localprovider/components/tEXistConnection/tEXistConnection_java.xml]: Fixed

**Review by mbasiuk-talend — APPROVED** [review]
approved new changes.

**Pull request merged**